zrml/prediction-markets/src/lib.rs

```diff
@@ -2347,6 +2412,12 @@ mod pallet {
                         err
                     );
                 }
+
```

Comment:
Should we use `repatriate` here, as well?

Reply:
Yeah indeed, we should!

bf6920c52d92d11884b406e0cc113368616e759e